src/actinia_core/resources/ephemeral_processing.py

```diff
@@ -1165,7 +1270,17 @@ def _run_executable(self, process, poll_time=0.005):
             self.module_output_dict[process.id] = plm
 
         if proc.returncode != 0:
-            raise AsyncProcessError("Error while running executable <%s>" % process.executable)
+            raise AsyncProcessError(
+                "Error while running executable <%s>" % process.executable)
+
+        # save interim results
+        if (self.save_interim_results is True
+                and self.temp_mapset_path is not None):
+            self._save_interim_results()
+        elif self.temp_mapset_path is None:
+            self.message_logger.debug(
+                "No temp mapset path set. Because of that no interim results" \
```

Comment:
Suggestion: please add where to set this (config?):

"No temp mapset path set in ADDME ..."

Reply:
`self.temp_mapset_path` is not set for endpoint like `locations`, `locations/{LOCATION}/mapsets`, ... .
I only added the debug message for us if the interim results are not saved/saved but we expect something other.

Reply:
Right - I overlooked that this is debugging only.